dev/cloudformation/grid-dev-core.yml

```diff
@@ -107,77 +124,20 @@ Resources:
         Projection:
           ProjectionType: ALL
         ProvisionedThroughput:
-          ReadCapacityUnits: '1'
-          WriteCapacityUnits: '1'
+          ReadCapacityUnits: 1
+          WriteCapacityUnits: 1
   IndexedImageTopic:
     Type: AWS::SNS::Topic
     Properties:
-      KmsMasterKeyId: !Sub "alias/media-service/sns-to-sqs-DEV"
```

Comment:
localstack doesn't support KMS aliases. KMS is used to encrypt a message in transit between SNS and SQS. Whilst best practice when running in AWS, it matters less in a localstack environment as everything is happening within a single Docker container on your local machine. So, rather than fight it, just remove KMS.